CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.0
+* New feature & breaking change: `noRecentsText` is now of type `Widget` (was `String` before). 
+  Inconsequence, `noRecentsStyle` was removed. This change leads to more customization options.
```

Comment:
Just small suggestion to match existing format
```suggestion
- Breaking Change: `noRecentsText` is now of type `Widget` (was `String` before) and `noRecentsStyle` was removed from Config
```
Eventually need to be adjusted if property is renamed.